src/test/java/io/jenkins/plugins/file_parameters/FileParameterWrapperTest.java

```diff
@@ -58,4 +62,129 @@
         r.assertLogContains("loaded 'UPLOADED CONTENT HERE' from ", b);
     }
 
+    @Ignore("need to implement option to tolerate undefined parameter")
+    @Test public void base64Undefined() throws Exception {
+        r.createSlave("remote", null, null);
+        WorkflowJob p = r.createProject(WorkflowJob.class, "myjob");
+        p.addProperty(new ParametersDefinitionProperty(new Base64FileParameterDefinition("FILE", null)));
+        String pipeline = "pipeline {\n" +
+            "  agent any\n" +
+            "  parameters {\n" +
+            "    base64File(name: 'FILE')\n" +
```

Comment:
You are not testing the simplified syntax? How do you know it works?

Reply:
simplified syntax doesn't work